mingus/midi/midi_file_in.py

```diff
@@ -332,6 +355,12 @@ def parse_midi_event(self, fp):
                 raise IOError("Couldn't read MIDI event parameters from file.")
             param1 = self.bytes_to_int(param1)
             param2 = self.bytes_to_int(param2)
+
+            # "It is common for a note on with 0 velocity to be interpreted as NOTE OFF." https://stackoverflow.com/questions/48687756/midi-note-on-event-without-off-event
+            # this should resolve Lilypad midi files which don't seem to use ec 8 to signify NOTE OFF
```

Comment:
Nice one. Did you mean LilyPond instead of Lilypad?

Reply:
Yes I did :)